v23.1/restore.md

```diff
@@ -208,7 +209,7 @@ In general, two types are compatible if they are the same kind (e.g., an enum is
 
 ### Object dependencies
 
-Dependent objects must be restored at the same time as the objects they depend on.
+Dependent objects must be restored at the same time as the objects they depend on. Referenced UDFs are not restored and require the [`skip_missing_udfs`](#skip_missing_udfs) option. 
```

Comment:
I suggest moving this new sentence below the table here. Since the existing sentence really helps to set context for the table following it, I think they should follow one another.